tools/deb/istio.mk

```diff
@@ -49,15 +50,19 @@ ISTIO_DEB_NAME ?= istio-sidecar
 # since we need configuration.
 # --iteration 1 adds a "-1" suffix to the version that didn't exist before
 ${ISTIO_OUT}/istio-sidecar.deb: | ${ISTIO_OUT}
+	$(MAKE) deb/fpm
```

Comment:
I'm not clear on why this is recursively calling make (vs. being a dependency), nor why this & deb/build-in-docker are using a name that looks like a directory path.  The "deb/fpm" (or whatever it ends up being called) can be added to the .PHONY list.

How was "make deb" insufficient for the debian build (and thus provoking the addition of deb/fpm?).  If it's about trying to bypass the dependencies (to avoid a possible rebuild attempt of the go binaries), then I suppose a possible option is to have deb/build-in-docker pass a flag to make that's used to disable a line like:

$(eval ${ISTIO_OUT}/istio-sidecar.deb: $(ISTIO_OUT)/$(DEP))

(might need to put the two $(eval) lines into their own foreach loop to avoid getting too hard to read).

Reply:
I'm trying to avoid and undo the evals - I think they are hard to read, I would rather have an
explicit clear rule.

In particular the 'build in docker' stuff doesn't seem to work anymore - all the magic with git seems to have affected it, will need to fix that as well.